rep-2000.rst

```diff
@@ -81,10 +81,91 @@ Package manager use for dependencies:
 
 Build System Support:
 
-- ament
+- ament_cmake
 - cmake
 - setuptools
 
+Middleware Implementation Support:
+
+- eProsima Fast-RTPS
+- RTI Connext
+- ADLink OpenSplice
+
+
+Bouncy Bolson (June 2018 - June 2019)
+----------------------------------------------
+
+Binary packages provided:
+
++----------------------------------------------------------------------------------------------+--------------------------------------------------+
+|                                     Required Support                                         |                Recommended Support               |
++--------------+-----------------------+------------------------------+------------------------+---------------------------+----------------------+
+| Architecture | Ubuntu Bionic (18.04) |     MacOS Sierra (10.12)     |   Windows 10 (VS2017)  |    Ubuntu Xenial (16.04)  |   Debian Stretch (9) |
++==============+=======================+==============================+========================+===========================+======================+
+|    amd64     |          X            |             X                |          X             |             N/A           |         N/A          |
++--------------+-----------------------+------------------------------+------------------------+---------------------------+----------------------+
+|    arm64     |          X            |                              |                        |             N/A           |         N/A          |
++--------------+-----------------------+------------------------------+------------------------+---------------------------+----------------------+
```

Comment:
What is this table supposed to express, "required and recommended binary support" or "required and recommend support building from source"?

Is it not a requirement for Bouncy to build from source on Xenial? Or are we softening that?

Reply:
> What is this table supposed to express, "required and recommended binary support" or "required and recommend support building from source"?


This is supposed to reflect what combinations of plaforms/architecture OSRF provides binaries for. Maybe the title `Binary packages provided:` should be highligted to make it clearer.
This is why the platforms where we don't planned to ever provide binaries are marked "N/A"

I should also propagate the `Recommended Support: source only` to this tables header to make it more explicit.

> Is it not a requirement for Bouncy to build from source on Xenial? Or are we softening that?

I'm not sure how to preperly express this, for example, it's already impossible to compile some of our packages on both platforms because of major changes in system dependencies (e.g. cartographer).

Maybe moving Xenial to the "Required support" section with a "source only" label allows to carry that "the ROS2 core should build on this platform"?

Reply:
I added https://github.com/ros-infrastructure/rep/pull/174/commits/40b67caeea4418446641460e6deda6d044def9be to try to clarify